packages/react-property/scripts/build-html.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs');
+const path = require('path');
+const DOMProperty = require('react-dom/lib/DOMProperty');
+const HTMLDOMPropertyConfig = require('react-dom/lib/HTMLDOMPropertyConfig');
+const { LIB_DIR, HTML_DIR } = require('./constants');
+
+/**
+ * Creates the DOM property map via injection.
+ *
+ * @see https://github.com/facebook/react/blob/15-stable/src/renderers/dom/shared/DOMProperty.js#L57
+ *
+ * ```js
+ * {
+ *   properties: {
+ *     accept: {
+ *       attributeName: 'accept',
+ *       attributeNamespace: null,
+ *       propertyName: 'accept',
+ *       mutationMethod: null,
+ *       mustUseProperty: false,
+ *       hasBooleanValue: false,
+ *       hasNumericValue: false,
+ *       hasPositiveNumericValue: false,
+ *       hasOverloadedBooleanValue: false
+ *     },
+ *     // ...
+ *   },
+ *   getPossibleStandardName: {
+ *     autofocus: 'autoFocus',
+ *     accept: 'accept',
+ *     acceptcharset: 'acceptCharset',
+ *     'accept-charset': 'acceptCharset',
+ *     // ...
+ *   },
+ *   // ...
+ * }
+ * ```
+ */
+DOMProperty.injection.injectDOMPropertyConfig(HTMLDOMPropertyConfig);
+
+/**
+ * Create output directories (if it does not exist).
+ */
+try {
+  fs.mkdirSync(LIB_DIR);
+} catch (error) {
+  // throw error;
```

Comment:
Should the error be re-thrown here if it is not a "directory already exists" error? 
```
if (error.code !== 'EEXIST') throw error;
```

Reply:
1566b2f